static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -1,34 +1,50 @@
 # Updating pipelines
 
-If you need to rename or relocate the file paths and/or dependencies of a
-specific DVC-file, then you can 
+If you need to rename or relocate the file paths and/or dependencies and
+outputs of a specific DVC-file, then you can
 
 - use `dvc move` to edit file paths of outputs or dependencies.
```

Comment:
dvc move also moves the file

Reply:
I think 'relocate' means just the same as 'move'